### ISSUE TITLE
Getting Cannot read property 'isReactComponent' of undefined

### ISSUE THREAD

**Issue by danieltott** [tannerlinsley/react-table#1502]
# Using v6?

Yes, this is in v6. 

**Describe the bug**
It looks like [this commit](https://github.com/tannerlinsley/react-table/commit/ab12816799de4c9ad2a970a15c9cc8925010cc7d) that was published in 6.10.1 re-introduced an issue with arrow functions and `isClassComponent` that was solved in #109 - the problem is that arrow functions do not have `.prototype` property. It seems like the solution in that case will work here: using `Object.getPrototypeOf(component).isReactComponent` instead of `component.prototype.isReactComponent` in `isClassComponent`

**To Reproduce**
1. Create default CRA app
2. Install `yarn add react-table`
3. `yarn start`
4. use the following code for App.js:
  ```jsx
import React from 'react';
import ReactTable from 'react-table';

const data = [
  {
    id: 1,
    text: 'One'
  },
  {
    id: 2,
    text: 'Two'
  }
];

const columns = [
  {
    Header: 'ID',
    accessor: 'id'
  },
  {
    Header: 'Text',
    Cell: row => <span>{row.original.text}</span>
  }
];

const App = () => (
  <div className="App">
    <ReactTable data={data} columns={columns} />
  </div>
);

export default App;
```

**Expected behavior**
Custom column Cells work using arrow functions

**Codesandbox!**
Interestingly, this behavior is not present in Codesandbox. I have no idea why.

If you change the arrow function to a traditional `function(){}`, it works fine.

I will give a PR a shot if that'd be the way to go.

**Comment by danieltott** [comment]
As a note until the PR gets merged/released, reverting to `6.10.0` resolves this specific issue (although may introduce other issues).

**Comment by tannerlinsley** [comment]
6.10.2 is now available with the fix. Thanks!
